Source/WebCore/Modules/webauthn/AuthenticationExtensionsClientInputs.h

```diff
@@ -27,14 +27,22 @@
 
 #if ENABLE(WEB_AUTHN)
 
+#include "BufferSource.h"
 #include <wtf/text/WTFString.h>
 
 namespace WebCore {
 
 struct AuthenticationExtensionsClientInputs {
+    struct LargeBlobInputs {
+        std::optional<String> support;
```

Comment:
We don't use `std::optional<String>`. String already has a null state so String::isNull() will be true when the support member wasn't provided.